packages/react/src/ssg/index.ts

```diff
@@ -0,0 +1,4 @@
+export { createSSGHelpers } from './ssg';
```

Comment:
Deprecate this

Reply:
The old one is still needed to use with old, "string-keyed" procedures

Reply:
Yes, but we should have a deprecation warning when you do

Reply:
Merging as is and moved my comment to #2450